pages/common/pnpm-outdated.md

```diff
@@ -0,0 +1,37 @@
+# pnpm outdated
+
+> Checks for outdated packages.
+> The check can be limited to a subset of the installed packages by providing arguments (patterns are supported).
+> More information: <https://pnpm.io/cli/outdated>.
+
+- Check for outdated packages:
+
+`pnpm outdated`
+
+- Check for outdated dependencies in every package found in every workspace package:
```

Comment:
```suggestion
- Check for outdated dependencies in every package found in every workspace:
```
Is this better?

or

```suggestion
- Check for outdated dependencies found in every workspace package:
```

Reply:
thank you, done ✅ I committed the second suggestion:
- Check for outdated dependencies found in every workspace package: